Interstation-Two-WW2/code/modules/organs/wound.dm

```diff
@@ -129,16 +132,22 @@
 		if (damage_type == BRUISE && !bleeding()) //bruises only infectable if bleeding
 			return 0
 
-		var/dam_coef = round(damage/10)
-		switch (damage_type)
-			if (BRUISE)
-				return prob(dam_coef*5)
-			if (BURN)
-				return prob(dam_coef*10)
-			if (CUT)
-				return prob(dam_coef*20)
+		return 1
+
+	// checks if wound is considered open for external infections
+	// untreated cuts (and bleeding bruises) and burns are possibly infectable, chance higher if wound is bigger
+	proc/infection_check()
+		if (src.can_be_infected())
+			var/dam_coef = round(damage/10)
+			switch (damage_type)
+				if (BRUISE)
+					return prob(dam_coef*5)
+				if (BURN)
+					return prob(dam_coef*10)
+				if (CUT)
+					return prob(dam_coef*20)
 
-		return 0
+	return 0
```

Comment:
Why would you want this outside of the proc? The whole point is to return 0 if nothing else is returned